src/i18n/locales/nl/translations.json

```diff
@@ -75,6 +75,11 @@
       "recommendedAndMandatoryRoutes": "Aanbevolen en verplichte routes",
       "displayOfRecommendedAndMandatoryRoutes": "Weergave van de aanbevolen en verplichte routes voor touringcars.",
```

Comment:
Aanpassing aangezien deze tekst altijd getoont wordt, niet alleen bij de agenda items 'aanbevolen' en 'verplichte', dus moet generieker
"recommendedAndMandatoryRoutes": "Situatie vanaf januari 2024",
      "displayOfRecommendedAndMandatoryRoutes": "Parkeerplaatsen, in- en uitstaphaltes en de verplichte touringcarroute in Amsterdam.",
"payAttentionToVerhicleHeights": "Let op de doorrijhoogtes.",

Reply:
Adapted as suggested